### PR TITLE
Added links to best practices page on package upload page

### PR DESCRIPTION
**Pull request by sophiamfavro**
Added links to best practices page in upload package page

Addresses https://github.com/NuGet/NuGetGallery/issues/8411

image

### CODE REVIEW

**Comment by dnfgituser** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/NuGet/NuGetGallery?pullRequest=8542) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/NuGet/NuGetGallery?pullRequest=8542) before we can accept your contribution.<br/><br/>:x: sophiamfavro [sign now](https://cla.dotnetfoundation.org/NuGet/NuGetGallery?pullRequest=8542)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/NuGet/NuGetGallery?pullRequest=8542) it.</sub>

File: src/NuGetGallery/Views/Packages/UploadPackage.cshtml [hunk 1]
```diff
@@ -17,7 +17,11 @@
             @ViewHelpers.AjaxAntiForgeryToken(Html)
             @ViewHelpers.PackagesBreadcrumb(Url, CurrentUser, true, @<text>Upload</text>)
             <div class="text-left">
-                <p class="message">Your package file will be uploaded and hosted on the @(Config.Current.Brand) server (@(Config.Current.SiteRoot)).</p>
+                <p class="message">Your  package file will be uploaded and on the @(Config.Current.Brand) server (@(Config.Current.SiteRoot)).</p>
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/UploadPackage.cshtml:20]: It looks like an extra space was added when we were playing around with this text:

```suggestion
                <p class="message">Your package file will be uploaded and on the @(Config.Current.Brand) server (@(Config.Current.SiteRoot)).</p>
```

File: src/NuGetGallery/Views/Packages/UploadPackage.cshtml [hunk 2]
```diff
@@ -52,18 +56,18 @@
                     </div>
                 </div>
             </div>
-
-            @if (Model != null && Model.IsUploadInProgress)
+            @* @if (Model != null && Model.IsUploadInProgress)*@
+            @if (true)
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/UploadPackage.cshtml:60]: Oops we forgot to undo our test code:

```suggestion
            @if (Model != null && Model.IsUploadInProgress)
```

File: src/NuGetGallery/Views/Packages/UploadPackage.cshtml [hunk 3]
```diff
@@ -52,18 +56,18 @@
...
             {
-                <div id="warning-container">
-                    @ViewHelpers.AlertInfo(@<text>You had an upload in progress. You can continue it here or cancel to restart.</text>)
-                </div>
+            <div id="warning-container">
+                @ViewHelpers.AlertInfo(@<text>You had an upload in progress. You can continue it here or cancel to restart.</text>)
+            </div>
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/UploadPackage.cshtml:64]: It looks like Visual Studio messed with the whitespace here. We can fix that tomorrow :)

File: src/NuGetGallery/Web.config [hunk 4]
```diff
@@ -321,7 +321,7 @@
   -->
   <system.web>
     <httpCookies requireSSL="true" httpOnlyCookies="true"/>
-    <compilation debug="false" targetFramework="4.7.2">
+    <compilation debug="true" targetFramework="4.7.2">
```
Comment (drewgillies) [inline src/NuGetGallery/Web.config:324]: This change needs to be discarded, right?

**Review by drewgillies — APPROVED** [review]
You've noted the formatting and test code changes still required. There's also `web.config`. Otherwise :shipit:

File: src/NuGetGallery/Web.config (hunk 4 above)
Comment (sophiamfavro) [reply src/NuGetGallery/Web.config:324]: Thank you for catching that!

**Review by loic-sharma — APPROVED** [review]

**Pull request merged**